src/types/zetkin.ts

```diff
@@ -207,3 +208,11 @@ export interface ZetkinTask {
         title: string;
     };
 }
+
+export interface ZetkinTaskReqBody extends Partial<ZetkinTask> {
+    title: string;
+    instructions: string;
+    type: ZetkinTaskType;
+    campaign_id: number;
+    target_filters: Array<unknown>; // What could they be?
```

Comment:
The `target_filters` array is the same as the `filter_spec`, i.e. a list of Smart Search filters.

Reply:
OK great, will make a type for those 👍